src/main/java/uk/ac/ebi/embl/api/validation/dao/EraproDAOUtilsImpl.java

```diff
@@ -192,7 +213,16 @@ public Reference getSubmitterReference(String analysisId) throws SQLException, U
 		
 		return reference;
 	}
-	
+
+	private String getAddress(ResultSet rs) throws SQLException, UnsupportedEncodingException {
+		return EntryUtils.concat(", ",
+				EntryUtils.convertNonAsciiStringtoAsciiString(rs.getString("center_name")),
+				EntryUtils.convertNonAsciiStringtoAsciiString(rs.getString("laboratory_name")),
+				EntryUtils.convertNonAsciiStringtoAsciiString(rs.getString("address")),
+				EntryUtils.convertNonAsciiStringtoAsciiString(rs.getString("country")));
```

Comment:
- For brokers, we need to use broker_name instead of center_name
- laboratory_name should only be part of address for non-brokers